Procfile.dev

```diff
@@ -1,4 +1,4 @@
-web: bin/rails server -p 3001
+web: RUN_SIDEKIQ_IN_TEST_MODE=false bin/rails server -p 3001
```

Comment:
Think this is unnecessary - by default, RUN_SIDEKIQ_IN_TEST_MODE is not set, and in your initializer, if it's not set, it defaults to false. So we'd only want to set this like this if we'd set RUN_SIDEKIQ_IN_TEST_MODE to true somewhere.

Reply:
could set it to nothing or anything other than 'true', but this was the only thing that seemed to get my sidekiq stats actually showing up and not running inline using `bin/dev`

Reply:
I guess it's just the inconsistency that .env.development in caseworker has `RUN_SIDEKIQ_IN_TEST_MODE=true`, so that needs to be overridden in the procfile to stop it happening, whereas in the provider app that line isn't there, so by default in develop mode sidekiq won't run in test mode, so overriding it from nil to "false" won't affect the logic in the initialiser either way.

Reply:
i wonder if there is some shell variable leakage going between the two as well, not too mention sidekiq job processing leakage, as fixing it for caseworker seemed to get stats showing up on provider...i think.

Anyway, I have amended the commit so it defaults to inline true for  typical `rails s` but then this switches it of for `bin/dev`  - this is at least consistent behaviour with caseworker (following my PR on that repo), and gets bin/dev doing what i would hope it does.